content/pages/home.mdx

```diff
@@ -26,27 +26,27 @@ beforeBody:
     _template: Carousel
   - bigCardsLabel: Consulting
     bigCards:
-      - title: Web / Angular / React
+      - title: Mobile / Bots / AI / ChatGPT
         description: >
-          Ever since SSW developed the very first .NET Website in Australia,
-          we've been known as leaders in Web  Development.
-        link: /consulting/web-applications
+          SSW software developers are experts at building cross platform mobile
+          and chat apps using the power of Al with Machine Learning.
+        link: /consulting/mobile-application-development
         color: red
-        imgSrc: /images/services/Angular.png
+        imgSrc: /images/services/Mobile.png
       - title: DevOps & Cloud
         description: >
           Make sure your software ships by using SSW developers' best practices
           in Azure, DevOps and Docker Containers.
         link: /consulting/azure
         color: lightgray
         imgSrc: /images/services/Cloud.png
-      - title: Mobile / Bots / AI
+      - title: Web / Angular / React
         description: >
-          SSW software developers are experts at building cross platform mobile
-          and chat apps using the power of Al with Machine Learning.
-        link: /consulting/mobile-application-development
+          Ever since SSW developed the very first .NET Website in Australia,
+          we've been known as leaders in Web  Development.
+        link: /consulting/web-applications
```

Comment:
Do you think it should be redirected to ChatGPT page instead of web-applications?